Resources/Prototypes/Entities/Clothing/Ears/headsets.yml

```diff
@@ -13,6 +13,18 @@
       key_slots:
       - EncryptionKeyCommon
   - type: Headset
+    sound: /Audio/Items/headset.ogg
```

Comment:
100% required variation + sound collection, because this sound can be played very often